tests/phpunit/admin/class-wc-rest-stripe-settings-controller-test.php

```
@@ -759,56 +759,64 @@ public function stripe_payment_method_configurations_provider() {
+		// Storage is a single location => methods map in canonical order.
+		$stored = $this->get_gateway()->get_option( 'express_checkout_button_locations' );
+		$this->assertSame( [ 'amazon_pay', 'link', 'payment_request' ], $stored['product'] );
+		$this->assertSame( [ 'payment_request' ], $stored['cart'] );
+		$this->assertSame( [ 'link' ], $stored['checkout'] );
+
+		// GET exposes each method's flat locations derived from the map.
+		$data = $this->rest_get_settings()->get_data();
+		$this->assertSame( [ 'product', 'cart' ], $data['express_checkout_button_locations'] );
+		$this->assertSame( [ 'product', 'checkout' ], $data['link_button_locations'] );
+		$this->assertSame( [ 'product' ], $data['amazon_pay_button_locations'] );
+	}
```

PHPMD (2.15.0)

[error] The method test_express_checkout_button_locations_unify_into_map is not named in camelCase. (undefined)

(CamelCaseMethodName)